### PR TITLE
Setup CI

### PR DESCRIPTION
**Pull request by nunocosta-casafari**
# Description

Setup Travis to test the package.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.mypy_cache
+.tox
+__pycache__
+**.egg-info
+**.pyc
```
Comment (dmytrostriletskyi) [inline .gitignore]: Use already existed one from any project which is full and you don't need to extend it in the future.

Comment (nunocosta-casafari) [reply .gitignore]: done

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+language: python
+python: "3.7"
+install: pip install tox-travis
+script: tox
```
Comment (dmytrostriletskyi) [inline .travis.yml]: Can we do it a more extendable way?

```yaml
language:
  - python

sudo:
  - false
  
python:
  - 3.7
  
before_install:
  - pip install tox-travis

script:
  - tox
```

File: test_hello_world.py [hunk 3]
```diff
@@ -0,0 +1,5 @@
+from hello_world import hello_world
+
+
+def test_hello_world():
+    assert 'Hello world!' == hello_world()
```
Comment (dmytrostriletskyi) [inline test_hello_world.py]: Can you move this to a folder? It's easier than to detect source code from many configuration files.

File: tox.ini [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# tox (https://tox.readthedocs.io/) is a tool for running tests
+# in multiple virtualenvs. This configuration file will run the
+# test suite on all supported python versions. To use it, "pip install tox"
+# and then run "tox" from this directory.
+
```
Comment (dmytrostriletskyi) [inline tox.ini]: Remove this template comment, please.

File: tox.ini [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+[tox]
+envlist = py37
```
Comment (dmytrostriletskyi) [inline tox.ini]: Can we extend this? From `Python` 3.4 (I think) to the latest one, 3.8.

File: .travis.yml (hunk 2 above)
Comment (nunocosta-casafari) [reply .travis.yml]: done

File: test_hello_world.py (hunk 3 above)
Comment (nunocosta-casafari) [reply test_hello_world.py]: done

File: tox.ini (hunk 4 above)
Comment (nunocosta-casafari) [reply tox.ini]: done

File: tox.ini (hunk 5 above)
Comment (nunocosta-casafari) [reply tox.ini]: done

File: .travis.yml [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+
+python: 
+    - "3.7"
+
+install:
+    - pip install tox-travis
+
+script: 
+    - tox
+
```
Comment (dmytrostriletskyi) [inline .travis.yml]: Excess blank line.

File: .travis.yml [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+
```
Comment (dmytrostriletskyi) [inline .travis.yml]: Excess blank lines.

File: .travis.yml (hunk 6 above)
Comment (nunocosta-casafari) [reply .travis.yml]: done

File: .travis.yml (hunk 7 above)
Comment (nunocosta-casafari) [reply .travis.yml]: done

**Pull request merged**